File: yukicoder/516_赤と青の風船/Main.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int redCount = 0;
        int blueCount = 0;

        for (int i = 0; i < 3; i++) {

            String color = sc.next();

            if (color.equals("RED")) {
                redCount++;
            } else if (color.equals("BLUE")) {
                blueCount++;
            }

        }

        System.out.println(redCount > blueCount ? "RED" : "BLUE");

    }

}

File: yukicoder/516_赤と青の風船/Main.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {

    int redCount = 0;
    int blueCount = 0;

    for (int i = 0; i < 3; i++) {

        string color;
        cin >> color;

        if (color == "RED") {
            redCount++;
        } else if (color == "BLUE") {
            blueCount++;
        }

    }

    if (redCount > blueCount) {
        cout << "RED" << endl;
    } else {
        cout << "BLUE" << endl;
    }

    return 0;

}
